### PR TITLE
navbar-component

### PR DESCRIPTION
**Pull request by Czaroos**
Will be rwd ready when icon button component is done.

### CODE REVIEW

**Review by i0skar — APPROVED** [review]

File: src/components/button/index.tsx [hunk 1]
```diff
@@ -8,10 +8,16 @@ export const Button = ({
...
-    <ButtonContainer onClick={onClick} variant={variant} {...props}>
+    <ButtonContainer
+      onClick={onClick}
+      variant={variant}
+      width={width}
+      {...props}
+    >
       {children}
     </ButtonContainer>
   );
```
Comment (ajronn) [inline src/components/button/index.tsx:23]: you should create other branch for fix button

File: tsconfig.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "paths": {
       "@fonts/*": ["/assets/fonts/*"],
       "@styles/*": ["/styles/*"],
-      "@components": ["/components"]
+      "@components": ["/components"],
+      "@images/*": ["/assets/images/*"]
     },
   },
   "include": ["./src"],
```
Comment (ajronn) [inline tsconfig.json]: same as above, other branch for fix config...

**Review by ajronn — APPROVED** [review]

File: src/components/button/index.tsx (hunk 1 above)
Comment (Czaroos) [reply src/components/button/index.tsx:23]: This customization wasn't needed until creating navbar component - that's why it's in this pr and this branch

File: tsconfig.json (hunk 2 above)
Comment (Czaroos) [reply tsconfig.json]: This will be resolved in conflict, no worries :)

**Pull request merged**